documentation/sphinx/source/api-version-upgrade-guide.rst

```diff
@@ -17,7 +17,17 @@ API version 710300
 General
 -------
 
-* Special keys ``\xff\xff/management/profiling/<client_txn_sample_rate|client_txn_size_limit>`` are removed in 7.2 and the functionalities they provide are now covered by the global configuration module.
+* No changes are needed
```

Comment:
In addition to the change below, there are other changes that happen in API version 710300 that we should capture